llvm/lib/Transforms/Instrumentation/RealtimeSanitizer.cpp

```diff
@@ -45,6 +46,26 @@ static void insertCallAtAllFunctionExitPoints(Function &Fn,
         insertCallBeforeInstruction(Fn, I, InsertFnName);
 }
 
+static PreservedAnalyses rtsanPreservedCFGAnalyses() {
+  PreservedAnalyses PA;
+  PA.preserveSet<CFGAnalyses>();
+  return PA;
+}
+
+static void insertNotifyBlockingCallAtFunctionEntryPoint(Function &F) {
```

Comment:
In the other function, this param is called "Fn" (for better or worse, but I think it would be good to be self consistent)

Reply:
Cool, sounds sensible - changed it to `Fn` throughout the file